cmd/kata-manager/kata-manager.sh

```diff
@@ -10,28 +10,38 @@ set -e
 
 typeset -r script_name=${0##*/}
 
-typeset -r doc_repo="github.com/kata-containers/documentation"
-typeset -r test_repo="github.com/kata-containers/tests"
+typeset -r kata_git_base="github.com/kata-containers"
+typeset -r doc_repo="${kata_git_base}/documentation"
+typeset -r test_repo="${kata_git_base}/tests"
+typeset -r tarball_suffix="/archive/master.tar.gz"
 
 typeset -r config_file_name="configuration.toml"
 typeset -r default_image_dir="/usr/share/kata-containers"
 typeset -r default_image_file="${default_image_dir}/kata-containers.img"
 typeset -r default_initrd_file="${default_image_dir}/kata-containers-initrd.img"
 typeset -r default_config_file="/usr/share/defaults/kata-containers/${config_file_name}"
 typeset -r local_config_file="/etc/kata-containers/${config_file_name}"
+typeset -r kata_doc_to_script="${test_repo}/.ci/kata-doc-to-script.sh"
+typeset -r -A dowloaders_list=([curl]="-fsSL" [wget]="-O -")
```

Comment:
cute! :)

Nit: Minor typo though as this should probably be `downloaders_list` (missing `n`)?